src/methods.jl

```diff
@@ -33,10 +35,14 @@ end
 
 
 """
-    subtract_bias(frame::AbstractArray, bias_frame::AbstractArray)
+    subtract_bias(frame, bias_frame; [hdu = (1, 1)])
 
 Subtract the `bias_frame` from `frame`.
 
+If either are strings, they will be loaded into `CCDData` first. The HDU loaded can be specified by `hdu` as either an integer or a tuple corresponding to each file.
+
+Function output inherits the type of first parameter and header file of output (if applicable) is same as of the first parameter.
```

Comment:
This reads a little confusing- I'm not sure it's really necessary for methods like `subtract_bias` where there is a clear "frame of interest" and a "calibration frame". Ideally, through the function name and the argument names it should be clear what is happening (I think so, although I may be wrong), and everything should just work.

Reply:
Since this is not the in-place version, things can be a bit ambiguous about the header.

User thoughts:
1. I am allocating a new `CCDData`, it will use the constructor `get_default_header` to get the header of the newly created struct.
2. I am allocating a new `CCDData`, it will have the same header as the frame of interest.

Therefore, I explicitly mentioned the source of the new header created. Thoughts?

Reply:
When a user goes for a non-mutating function, the method creates a new instance of struct `CCDData`, right?

Now, how will the user know, whether the header which was in output (new struct) was copied from "calibration frame" or generated using the `get_default_header`?

Reply:
To tackle `(::Array, ::CCDData)` (output will be an `Array`) case, I have mentioned the applicability:
"header file of output (if applicable) is same as of the first parameter."

Reply:
that language is still really hard to parse, in my opinion! A different way of saying the same thing is "if `frame` is an `AbstractCCDData` the output will copy `frame`'s header". To me, this seems like the expected behavior, so it's weird to explicitly write out the documentation, but I guess it can't hurt.

Reply:
Ya, that looks much better than the current one!

Reply:
Okay, let's remove it from here and show some functionalities of `a + b` in `CCDData` struct docs, we can specify the header stuff over there.